### PR TITLE
fix: rename recycle into trash and add missing translations

### LINKED ISSUE #85: Change Recycle to Trash in english

### PR DESCRIPTION
**Pull request by spaenleh**
This PR updates some recycling related translations and adds missing translations in french.

closes #85

### CODE REVIEW

File: src/langs/fr/messages.json [hunk 1]
```diff
@@ -66,5 +71,26 @@
...
+  "UPDATE_PASSWORD": "Le mot de passe a été mis à jour avec succès",
+
+  "INVALID_ARCHIVE_FILE": "Le fichier soumis n'est pas une archive zip",
+  "INVALID_FILE_ITEM": "Les propriétés du fichier sont invalides",
+  "UNEXPECTED_EXPORT_ERROR": "Une erreur est survenue durant l'exportation de l'archive zip",
+
+  "NOT_APP_ITEM": "L'élément cible n'est pas une application",
+  "INVALID_APP_ORIGIN": "L'application ne correspond pas à l'origine enregistrée",
+  "TOKEN_ITEM_ID_MISMATCH": "Le jeton d'authentification ne correspond pas à l'élément cible",
+  "APP_DATA_NOT_FOUND": "Aucune données d'application trouvés",
```
Comment (dialexo) [inline src/langs/fr/messages.json:83]:
```suggestion
  "APP_DATA_NOT_FOUND": "Aucune donnée d'application trouvée",
```
or
```suggestion
  "APP_DATA_NOT_FOUND": "Aucunes données d'application trouvées",
```
though the latter is not technically correct (https://www.larousse.fr/dictionnaires/francais/aucun/6369)

File: src/langs/fr/messages.json [hunk 2]
```diff
@@ -66,5 +71,26 @@
...
+  "APP_DATA_NOT_ACCESSIBLE": "Le membre ne peut pas demander cette donnée d'application",
+  "APP_ACTION_NOT_ACCESSIBLE": "Le membre ne peut pas demander cette action d'application",
+  "APP_SETTING_NOT_FOUND": "Aucun paramètres d'application trouvés",
```
Comment (dialexo) [inline src/langs/fr/messages.json:86]: Similarly
```suggestion
  "APP_SETTING_NOT_FOUND": "Aucun paramètre d'application trouvé",
```
or
```suggestion
  "APP_SETTING_NOT_FOUND": "Aucuns paramètres d'application trouvés",
```

**Review by dialexo — CHANGES_REQUESTED** [review]
I don't if using the plural or singular form of "aucun" makes more sense... I've left a link to the Larousse page, i'll let you decide 🎲

**Comment by spaenleh** [comment]
Thank you for these suggestions @dialexo I choose the plural forms as they made more sens to me !

**Review by dialexo — APPROVED** [review]
Thank you for the changes 🇫🇷

**Pull request merged**